src/main/java/io/pravega/perf/ReaderWorker.java

```diff
@@ -69,12 +69,13 @@ public void benchmark() throws IOException {
         public void benchmark() throws IOException {
             String ret = null;
             try {
-                for (int i = 0; i < events; i++) {
+                for (int i = 0; i < events; ) {
```

Comment:
For clarity, maybe this needs to be a `while`.

Reply:
done